selfdrive/car/hyundai/hyundaican.py

```diff
@@ -119,6 +121,12 @@ def create_acc_commands(packer, enabled, accel, upper_jerk, idx, lead_visible, s
     "aReqValue": accel,  # stock ramps up and down respecting jerk limit until it reaches aReqRaw
     "CR_VSM_Alive": idx % 0xF,
   }
+
+  # send CF_VSM_ConfMode and AEB_Status to cars that do not accept FCA messages, otherwise TCS13|ACCEnable would fault
+  if not has_fca:
+    scc12_values["CF_VSM_ConfMode"] = 1
```

Comment:
know what `CF_VSM_ConfMode` does?

Reply:
It very likely stands for "Cruise Function | Vehicle Stability Management | Configuration Mode", but as far as what each bit means, we did not have a chance to test that. When the stock SCC system is in use, this signal stays at `2`.

Reply:
What led you to believe you should set it to 2?

Reply:
We tried setting this to 2 along with sending `AEB_Status = 1`, but it resulted in a fault.

Reply:
~What led you to believe you should set it to 1?~ @gregjhogan do you know how VSM plays with FCA/AEB? I see we can request braking via FCA, AEB or VSM